parser.go

```diff
@@ -102,6 +103,12 @@ func (p *Parser) Progress() float32 {
 	return float32(p.currentFrame) / float32(p.header.PlaybackFrames)
 }
 
+// GrenadeProjectiles returns a map with all grenade projectiles. The map contains only projectiles
+// that are currently in-flight, i.e. have been thrown but have yet to detonate.
+func (p *Parser) GrenadeProjectiles() map[int]*common.GrenadeProjectile {
```

Comment:
I think it might be preferable to put this inside `Parser.gameState` if possible.

The parser is already fairly cluttered and it would fit pretty well there.

Reply:
That's a good point!